include/LightGBM/cuda/cuda_row_data.hpp

```diff
@@ -0,0 +1,180 @@
+/*!
+ * Copyright (c) 2021 Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE file in the project root for license information.
+ */
+
+#ifdef USE_CUDA_EXP
+
+#ifndef LIGHTGBM_CUDA_ROW_DATA_HPP_
+#define LIGHTGBM_CUDA_ROW_DATA_HPP_
+
+#include <LightGBM/bin.h>
+#include <LightGBM/config.h>
+#include <LightGBM/cuda/cuda_utils.h>
+#include <LightGBM/dataset.h>
+#include <LightGBM/utils/openmp_wrapper.h>
+
+#include <vector>
+
+#include "../train_share_states.h"
```

Comment:
@shiyu1994 please update the include style.

Reply:
Most relative path include has been removed via f44b881. But there are still some 13 in the code base. Among them only 1 is introduced by this PR (https://github.com/microsoft/LightGBM/pull/4630#discussion_r820169671). It is not trivial to remove these relative path include since this may require either adding more paths to the include path, or moving these header files to `include/LightGBM`.

Reply:
![relative](https://user-images.githubusercontent.com/14541765/157381066-d3ab72e0-8290-46fd-877c-9ef9364d6604.PNG)